circle.yml

```diff
@@ -44,11 +46,17 @@ deployment:
   docker:
     branch: master
     commands:
-      # Build small deploy container for master branch.
       - sed "s/<EMAIL>/$DOCKER_EMAIL/;s/<AUTH>/$DOCKER_AUTH/" < "resources/deploy_templates/.dockercfg.template" > ~/.dockercfg
       - |
+          VERSION=$(git describe 2>/dev/null || git rev-parse --short HEAD 2>/dev/null)
```

Comment:
Perhaps we should also bake build information into the cockroach binary. We did this at Square and it was useful. I'll track down how that worked (something with go linker flags).